action.yml

```diff
@@ -44,6 +64,8 @@ runs:
         path: ".jit/"
 
     - name: Checkout original repository
+      # we don't want enrichment to fail the action on empty repo, it has its own logic to handle it
+      continue-on-error: true
       # this if should pass if there is not runner setup (old flow would checkout always) or when new flow ask to checkout
```

Comment:
1. is this new? i remember when no checkout thing should still work
2. what will happen on a valid asset when checkout fails?

Reply:
1. The underlying checkout action we used moved to V3 and introduced this erroring on empty repo while before that we didn't.
2. If a checkout will fail now it will propagate further now, and be delt with in the entrypoint, same as it did before checkout V3 was introduced